package.json

```diff
@@ -32,5 +32,8 @@
   },
   "browserslist": [
     "last 2 versions"
-  ]
+  ],
+  "dependencies": {
+    "gulp-cli": "^2.3.0"
+  }
```

Comment:
Gulp is a global dependency on these projects. It's necessary to run the build tools, but not to run the project, and therefore shouldn't be tracked as a "dependency."